.github/workflows/MainDistributionPipeline.yml

```diff
@@ -22,8 +22,8 @@ jobs:
 
   duckdb-stable-build:
     name: Build extension binaries
-    uses: duckdb/extension-ci-tools/.github/workflows/_extension_distribution.yml@v1.2.1
+    uses: duckdb/extension-ci-tools/.github/workflows/_extension_distribution.yml@main
     with:
       duckdb_version: v1.2.1
-      ci_tools_version: v1.2.1
+      ci_tools_version: main
       extension_name: zipfs
```

Comment:
is this intended to be set back to a versioned branch once the next release happens?

Reply:
I think given that community extensions default are:
* latest stable release for duckdb
* `main` for extension-ci-tools
and that changes are scarse, I would think main is OK (and I use `main` when setting up extensions myself), but I guess it's a matter of preferences, and also moving to `v1.2.2` do work.